truenas/documentation
# Pd 1089 fix header anchor i ds

To test this in a local build:

Go to http://localhost:1313/gettingstarted/scalereleasenotes/ and in a separate tab open https://www.truenas.com/docs/scale/24.04/gettingstarted/scalereleasenotes/. 

Click the toc link on both to "24.04-RC.1 Changelog. On the live site this directs you to `#24.04-rc.1-changelog`. The local build should correctly link to `2404-rc1-changelog`.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.
